docs/manual/podcasts/index.md

```diff
@@ -1,10 +1,34 @@
+# The Podcasts page
+
 The Podcasts page allows you add subscriptions to podcasts which are often used to syndicated audio files using a URL called a RSS feed. This allows your LibreTime instance to automatically download new shows from the web.
 
 In order to add a podcast you need to get the RSS feed. All podcasts available on iTunes have a RSS feed but it is sometimes hidden. See this issue on our github page [#510](https://github.com/LibreTime/libretime/issues/510) for more information. RSS feeds that do not end in *.xml* may be accepted by LibreTime but might fail to download episodes; in that case, download the episode using a podcast client such as [gpodder](https://gpodder.github.io/) and then manually upload and schedule the episode. Podcast feeds coming from Anchor.fm have been known to have this issue.
 
-The podcast interfaces provides you with the ability generate [Smartblocks](../smartblocks) that can be used in conjunction with [Autoloading Playlists](../calendar/#autoloading-playlist) to schedule the newest episode of a podcast without human intervention.
+The podcast interfaces provides you with the ability to generate [Smartblocks](../smartblocks) that can be used in conjunction with [Autoloading Playlists](../calendar/#autoloading-playlist) to schedule the newest episode of a podcast without human intervention.
 
 ##Video Tutorials##
 Coming Soon
```

Comment:
The video tutorials are here, so this Coming Soon can be removed

Reply:
I found this one: https://www.youtube.com/watch?v=g-4UcD8qvR8. Are there others?

Reply:
Thought there were more, but looking again, that one was all I could find too.

Reply:
It's a start...